src/main/java/eu/europeana/metis/sandbox/dto/DatasetInfoDto.java

```diff
@@ -28,14 +26,18 @@
   @JsonProperty("country")
   private final Country country;
 
+  @JsonProperty("xslt-transformed")
+  private final boolean xsltTransformed;
```

Comment:
All of the datasets are transformed to edm internal.
Are we descriptive enough here? What does anyone understand by reading this name?

Reply:
This field is supposed to indicate if together with the dataset a xslt file was given, that is, if the extra transformation step was performed or not. This is what Andy and I agreed to write in the report, he uses this field to let know the user that the given xslt file went through. If you believe this name is not explicit enough, I can always talk with Andy again about this and change it to a more descriptive name. What do you think?

Reply:
I get that. But that's a verbal contract between you and Andy. I as an outsider when I try to consume the api call or just read the code, will not understand what this means in particular. Especially because we transform twice. At the very least there should be some javadoc explaining this contract, but I'm in favor of having more descriptive names(in addition to the javadoc if needed).

Reply:
I understand, I'll handle this